examples/training/object_detection/pascal_voc/faster_rcnn.py

```diff
@@ -319,6 +324,7 @@ def pad_fn(examples):
 step = 0
 
 callbacks = [
+    keras.callbacks.EarlyStopping(patience=10),
```

Comment:
Instead of adding early stopping, let's just default the "epochs" flags to the current hard-coded default in both files.

Reply:
Done @ianstenbit